src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2460,6 +2487,14 @@ void processContextElement
       searchContextProviders(tenant, servicePathV, *enP, ceP->contextAttributeVector, cerP);
       cerP->statusCode.fill(SccOk);
       responseP->contextElementResponseVector.push_back(cerP);
+
+      //
+      // If no context providers found, then the UPDATE was simply for a non-found entity and an error should be returned
+      //
+      if (forwardsPending(responseP) == false)
```

Comment:
I understand that this is the change in mongoBackend that is causing that some 200 OK test pass to 404 Not found. I'd need some more of context of why this is needed to make an opinion...